ibis/pandas/core.py

```diff
@@ -220,7 +264,7 @@ def execute_with_scope(
             **kwargs,
         ),
         **kwargs,
-    )[op]
+    )[op]['value']
```

Comment:
What happens if the `value` here is for a context that is larger than the one that user requested?

Reply:
Checked again, this seems unlikely to happen because `execute_until_in_scope` would return with a dict with `op` as the only key. This is not reading from the scope. Time context should be already taken care of in execution phases in `execute_until_in_scope`

Reply:
I think `execute_until_in_scope` used to return the scope after execution. Is that still the case? If so, shouldn't the code here be:

```
result = get_scope_item(scope, op, timecontext)
```
?

Reply:
Yes it makes sense, however, `execute_until_in_scope` is a recursive call and it always returns the executed result of the expr, instead of the whole scope. Before this change, the return value is just `{op: computed}`.  So I think we don't need to call `get` method again here.

Reply:
ok